mapit.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [[ $(< /home/pi/network-conf) == "manual" || $(< /home/pi/network-conf) == "server" ]]; then
```

Comment:
This script could be one line:
`cat /home/pi/network-conf`

put 'config-manual' in the contents of network-conf

But let's add a few more to check if the file exists and if not, provide a default string